src/main/java/uk/gov/justice/hmpps/prison/service/OffenderAssessmentService.java

```diff
@@ -118,6 +120,22 @@ private AssessmentQuestion getAssessmentQuestionAndAnswers(final AssessmentEntry
             answers.stream().skip(1).collect(toList()));
     }
 
+    public List<AssessmentClassification> getOffendersAssessmentRatings(List<String> offenderList) {
```

Comment:
You mentioned above about dealing with large offender numbers, should we add an additional finder method that will return them all in one go? We'd have to batch them of course so comes with some complexity.

Reply:
Yes, I did consider it but I thought it wasn't a big enough gain for the work required. We are close to the DB here so making many calls to the DB doesn't have a large overhead compared to 1 call. And the memory consumption will be the same (if not better).

It's a good point though - now I am wondering if there might be a performance problem for lot of them...  I need to test it